src/styles/blocks/header.scss

```diff
@@ -0,0 +1,81 @@
+.header {
+    display: inline-flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 100%;
+    height: $header-height;
```

Comment:
Ensure that the `$header-height` variable is defined in your SCSS files. If it's not defined, it will cause a compilation error.